src/main/java/co/rsk/federate/signing/hsm/advanceBlockchain/HsmBookkeepingClientImpl.java

```diff
@@ -0,0 +1,200 @@
+package co.rsk.federate.signing.hsm.advanceblockchain;
```

Comment:
Please update the package name. Intellij is complaining cause the different cases of the word "blockchain".